NEWS.md

```diff
@@ -1,6 +1,25 @@
 # cmdstanr 0.8.1.9000
 
-Items for next release go here
+* Removed deprecated items (replacements in parentheses): 
```

Comment:
I made a list of the user-facing stuff that's being removed. I guess it's listed under `cmdstanr 0.8.1.9000` since that's the placeholder we use for the next release, but it's really for version 1.0